src/core/main.c

```diff
@@ -351,6 +352,16 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 else
                         arg_dump_core = r;
 
+        } else if (proc_cmdline_key_streq(key, "systemd.early_core_pattern")) {
+
+                if (proc_cmdline_value_missing(key, value))
+                        return 0;
+
+                if (path_is_absolute(value))
+                        parse_path_argument_and_warn(value, false, &arg_early_core_pattern);
+                else
+                        log_warning("Default core pattern is not an absolute path. Ignoring");
```

Comment:
i think this is a bit misleading, the default core pattern is `!/bin/false` after all...

Maybe:

```c
log_warning("Specified core pattern '%s' is not an absolute path, ignoring.", value);
```

Reply:
this remains unaddressed